generators/stats.js

```diff
@@ -52,12 +52,33 @@ Blockly.JavaScript['stats_t_test_one_sample'] = (block) => {
 // Create a paired two-sided t-test.
 //
 Blockly.JavaScript['stats_t_test_paired'] = (block) => {
+
   const order = Blockly.JavaScript.ORDER_NONE
   const left = block.getFieldValue('LEFT_COLUMN')
   const right = block.getFieldValue('RIGHT_COLUMN')
   const significance = block.getFieldValue('SIGNIFICANCE')
   const params = `{alpha: ${significance}}`
-  return `.test(${block.tbId}, environment, tbTTestPaired, ${params}, "${left}", "${right}")`
+  const suffix = TbManager.registerSuffix('')
```

Comment:
This makes me think that all of the Vega stuff should be moved into functions rather than being in the generated code, but we can do that cleanup later.